Modules/hpe3par_remote_copy.py

```diff
@@ -0,0 +1,1371 @@
+        
+  - name: Remove volume from remote copy group
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: remove_volume
+      remote_copy_group_name: farhan_rcg
+      volume_name: demo_volume_2
+      targets:
+      - targetName: CSSOS-SSA06
+        secVolumeName: demo_volume_2
+        
+  - pause:
+      prompt: "Press Enter"
+        
+  - name: Remove Remote Copy Group farhan_rcg
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: absent 
+      remote_copy_group_name: farhan_rcg
+
+  - pause:
+      prompt: "Press Enter"
+
+  - name: dismiss remote copy link
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: dismisslink
+      target_name: CSSOS-SSA06
+      source_port: 0:3:1
+      target_port_wwn_or_ip: 192.168.1.2
+
+  - name: dismiss remote copy link
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: dismisslink
+      target_name: CSSOS-SSA06
+      source_port: "1:3:1"
+      target_port_wwn_or_ip: 192.168.2.2
+
+  - pause:
+      prompt: "Press Enter"
+
+  - name: Admit remote copy link
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: admitlink
+      target_name: CSSOS-SSA06
+      source_port: 0:3:1
+      target_port_wwn_or_ip: 192.168.1.2
+
+  - name: Admit remote copy link
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: admitlink
+      target_name: CSSOS-SSA06
+      source_port: "1:3:1"
+      target_port_wwn_or_ip: 192.168.2.2
+
+  - pause:
+      prompt: "Press Enter"
+
+  - name: start remote copy service
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: startrcopy
+'''
+
+RETURN = r'''
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+try:
+    from hpe3par_sdk import client
+except ImportError:
+    client = None
+
+def validate_and_form_target(targets):
+    targets_transformed=[]
+    for dict1 in targets:
+        new_dict={}
+        for key in dict1.keys():
+            if key == 'target_name':
+                new_dict['targetName']=dict1.get(key)
+            if key == 'target_mode':
+                if dict1.get(key) == 'SYNC':
+                    new_dict['mode'] = 1
+                if dict1.get(key) == 'PERIODIC':
+                    new_dict['mode'] = 3
+                if dict1.get(key) == 'ASYNC':
+                    new_dict['mode'] = 4
```

Comment:
Are we handling invalid values anywhere?

Reply:
use else if... and finally use else if invalid values are provided and throw an error